plugin-server/src/utils/db/db.ts

```diff
@@ -540,6 +541,41 @@ export class DB {
         return client ? kafkaMessages : updatedPerson
     }
 
+    public async updatePersonPropertiesOld(
```

Comment:
copied from process-event.ts to make the comparison over the whole function for the old way too